core/scripts/local/DeployEMP.js

```diff
@@ -51,10 +54,12 @@ const deployEMP = async callback => {
   try {
     const accounts = await web3.eth.getAccounts();
     const deployer = accounts[0];
-    expiringMultiPartyCreator = await ExpiringMultiPartyCreator.deployed();
+    expiringMultiPartyCreator = await ExpiringMultiPartyCreator.at(
+      deployedAddresses.find(c => c.contractName === "ExpiringMultiPartyCreator").address
+    );
 
-    // Use Dai as the collateral token.
-    collateralToken = await TestnetERC20.deployed();
```

Comment:
I had previously removed my `build/` folder which meant I could no longer use the `.deployed()` truffle feature. I think changing to dynamically detecting the deployed address from `networks/` is more scalable.

Reply:
Ended up reverting this change because I can easily run `npm run apply-registry` to convert all addresses in `networks/` to artifacts